root.sh

```diff
@@ -227,8 +227,10 @@ module load BASE/1.0 ${ALIEN_RUNTIME_VERSION:+AliEn-Runtime/$ALIEN_RUNTIME_VERSI
 setenv ROOT_RELEASE \$version
 setenv ROOT_BASEDIR \$::env(BASEDIR)/$PKGNAME
 setenv ROOTSYS \$::env(ROOT_BASEDIR)/\$::env(ROOT_RELEASE)
-prepend-path PYTHONPATH \$::env(ROOTSYS)/lib
-prepend-path PATH \$::env(ROOTSYS)/bin
-prepend-path LD_LIBRARY_PATH \$::env(ROOTSYS)/lib
+
+set ROOTSYS  \$::env(ROOTSYS)
+prepend-path PYTHONPATH \$ROOTSYS/lib
```

Comment:
Please let's try to keep it uniform with the rest, why not using `\$ROOT_ROOT`?

Reply:
Well, as ROOTSYS was an env variable I just created a copy of it that is a normal variable. I could also rename this "normal ROOTSYS" variable to ROOT_ROOT. I'm on it.